scripts/start-setupVanillaTweaks

```diff
@@ -81,4 +83,20 @@ if [[ "$PACKS" ]] && [[ "$VT_VERSION" ]]; then
   fi
 fi
 
+# Download and move crafting tweaks
+if [[ "$CRAFTING_TWEAKS" ]] && [[ "$VT_VERSION" ]]; then
+  VT_ZIPDATA_URL=https://vanillatweaks.net/assets/server/zipcraftingtweaks.php
+    DOWNLOAD_URL=$(curl -X POST -F "packs=${CRAFTING_TWEAKS}" -F "version=${VT_VERSION}" $VT_ZIPDATA_URL | jq -r '.link')
+  if [ ! "$DOWNLOAD_URL" ]; then
+    log "ERROR: unable to retrieve crafting tweak DOWNLOAD_URL from vanillatweaks.net!"
+    exit 2
+  fi
+
+  mkdir -p "$out_dir"
+  if ! get -o $out_dir/crafting_tweaks.zip "https://vanillatweaks.net${DOWNLOAD_URL}"; then
```

Comment:
This is a zip of zips. Does this not need to be unzipped?

Reply:
Works fine locally 🤷 I checked the file downloaded and it's a single zipped directory, unlike the datapacks zip.

Reply:
Agreed, I did end up looking and that is a single zip. My apologies, thought it was like datapacks.